src/main/java/org/opensearch/knn/jni/FaissService.java

```diff
@@ -30,14 +32,17 @@
  *      src/main/java/org/opensearch/knn/index/query/KNNQueryResult.java
  *      src/main/java/org/opensearch/knn/common/KNNConstants.java
  */
+@SuppressWarnings("deprecation")
```

Comment:
Why was this needed?

Reply:
added by VSCode accidentally. Fixed with the latest commit.